content/pandas/dataframe/apply/apply.md

```diff
@@ -0,0 +1,133 @@
+
+## Syntax
+
+```pseudo
+x = dataframevalue.apply(func, axis=0, raw=False, result_type=None, args=(), **kwargs)
+```
+- `dataframevalue` is the DataFrame with the source data.
+- x means `return value`, a DataFrame or a Series object, with the changes.
+- This function does `NOT` make changes to the original DataFrame object. The result is saved to a variable, like `x` here.
+
+
+`DataFrame.apply()` has the following parameters:
+
+| Parameter Name | Data Type                | Usage                                                                                                                             |
+| :------------: | ------------------------ | --------------------------------------------------------------------------------------------------------------------------------- |
+|   `func`   |  function name     | Function to apply to each column or row.|
+|   `axis`   | 0/1 or 'index'/'columns', default 0 | Axis along which the function is applied: `0 or 'index'`: apply function to each column; `1 or 'columns'`: apply function to each row.|
+|   `raw`    | bool, default False | Determines if row or column is passed as a Series or ndarray object: `False`, passes each row or column as a Series to the function; `True`, it passes ndarray objects instead. This will achieve much better performance when applying a Numpy reduction function|
+| `result_type` | {'expand', 'reduce', 'broadcast', None}, default None | These only act when axis=1 (columns): `'expand'`, list-like results will be turned into columns; `'reduce'`, returns a Series if possible rather than expanding list-like results. This is the opposite of 'expand'; `'broadcast'`, results will be broadcast to the original shape of the DataFrame, the original index and columns will be retained; `'None'`, The default behaviour depends on the return value of the applied function.|
+|    `args`  | tuple   | Positional arguments to pass to func in addition.    |
+|  `**kwargs`  |     | Additional keyword arguments to pass as keywords arguments to func. |
+
+## Example
+
+In the following examples, the `.apply()` method is used with different parameters:
+
+```py
+import pandas as pd
+import numpy as np
+
+d = {'col 1' : [1,2,3,4], 'col 2' : [5,6,7,8], 'col 3' : [9,10,11,12], 'col 4' : [13,14,15,16]}
+
+df = pd.DataFrame(data = d)
+
+def calc_sum(x):
+  return x.sum()
+
+x = df.apply(calc_sum)
+y = df.apply(calc_sum,axis = 0)
+z = df.apply(calc_sum,axis = 1)
+l = df.apply(np.sqrt)
+m = df.apply(lambda x: pd.Series([0, 1, 2]), axis=1, result_type="expand")
+
+print(df)
+
+print(x)
+print(type(x))
+
+print(y)
+print(type(y))
+
+print(z)
+print(type(z))
+
+print(l)
+print(type(l))
+
+print(m)
+print(type(m))
+```
+Results are below:
+
+```shell
```

Comment:
The output block should be the actual output of the example. You're still missing the result of the first `print(df)`

Reply:
sure. Agree with you. I will update it basing on your suggest and pull request again a afew hours later